src/components/admin/agency/agency-profile-form.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles'; 
+import { Grid, TextField, Button, Box } from '@material-ui/core';
+import { Formik } from 'formik';
+import ImageOutlinedIcon from '@material-ui/icons/ImageOutlined';
+import LogoUpload from '../agency/logo-upload'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& > *': {
+      margin: theme.spacing(1),
+      width: '50ch'
+    },
+  },
+  errorMessage: {
+    color: 'red'
+  },
+  btn: {
+    textTransform: 'none',
+    width: '100%',
+    marginBottom: '3%'
+  }
+}));
+ 
+ export default function AgencyProfileForm() {
+  const classes = useStyles();
+
+  return (
+   <Grid container
+          spacing={0}
+          direction="column"
+          alignItems="center"
+          justify="center"
+          style={{ minHeight: '30vh' }}>
+     <Formik
+       initialValues={{ address: '', phone: '' }}
+       validate={values => {
+         const errors = {};
+         if (!values.phone) {
+           errors.phone = 'Required';
+         } else if (
+           !/^[+]{1}374{1}[0-9]{8}$/i.test(values.phone)
+         ) {
+           errors.phone = 'Invalid phone number';
+         }
+         return errors;
+       }}
+       onSubmit={(values, { setSubmitting }) => {
+         setTimeout(() => {
+           alert(JSON.stringify(values, null, 2));
+           setSubmitting(false);
+         }, 400);
+       }}
+     >
+       {({
+         values,
+         errors,
+         touched,
+         handleChange,
+         handleBlur,
+         handleSubmit,
+         isSubmitting,
+       }) => (
+        <Grid item xs={4}>
+         <form className={classes.root} onSubmit={handleSubmit}>
+           <TextField
+             type="text"
+             name="address"
+             onChange={handleChange}
+             onBlur={handleBlur}
+             value={values.email}
+             label="Address" 
+             variant="outlined" 
+             size ="small"
+           />
+           <Box className={classes.errorMessage}>{errors.address && touched.address && errors.address}</Box>
+           <TextField
+             type="tel"
+             name="phone"
+             onChange={handleChange}
+             onBlur={handleBlur}
+             value={values.phone}
+             label="Phone" 
+             variant="outlined" 
+             size ="small"
+           />
+           <Box className={classes.errorMessage}>{errors.phone && touched.phone && errors.phone}</Box>
+           
+           <Formik 
+            initialValues={{ file: null }}>
+              {({ values, setFieldValue }) =>(
+                <Box>
+                <input id='file' type='file' className={classes.btn} onChange={(event) => {
```

Comment:
Do not pass an arrow function as a callback, it will pass a new function each time. In functional components please use `useCallback` hook.